### PR TITLE
Communicating Widget States

### PR DESCRIPTION
**Pull request by zcorpan**

### CODE REVIEW

File: widget-states.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+aria-readonly
+
+
+HTML equivalents:
+
+```
+<input readonly>
+<input disabled>
+<input type=checkbox checked>
+<input type=checkbox>.indeterminate = true; <-> aria-checked = undefined
```
Comment (a11ydoer) [inline widget-states.md]: I understand what this means to some extent. Can we improve this a bit clear?

**Comment by mcking65** [comment]
ARIA 1.2 adds support for aria-expanded on roles application and checkbox. They need to be addressed in this set of changes since we will include this in the first draft of APG 1.2.

**Comment by zcorpan** [comment]
This section needs some dedicated work to finish. Unassigning for now.

**Comment by zcorpan** [comment]
@a11ydoer can you review this? Thanks!

**Comment by mcking65** [comment]
Simon, let's put this in a branch in the w3c repo; will make collaborating on the final edits easier.

**Comment by zcorpan** [comment]
@mcking65 Done: https://github.com/w3c/aria-practices/pull/1277

**Pull request closed without merging**